incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/KafkaGrpcBinding.java

```diff
@@ -55,6 +60,15 @@ public int workers(
     public KafkaGrpcBindingContext supply(
         EngineContext context)
     {
-        return new KafkaGrpcBindingContext(config, context);
+        return new KafkaGrpcBindingContext(config, context, this::canInitiate);
```

Comment:
See below, we need to add a decrement to cleanup the concurrent map.

Perhaps rename `canInitiate` to `activate` and add a `deactivate`.

```
private void deactivate(
    long bindingId)
{
    AtomicInteger worker = workers.get(bindingId);
    assert worker != null;
    worker.decrementAndGet();
    workers.remove(bindingId, new AtomicInteger(0)); // can be a constant
}
```